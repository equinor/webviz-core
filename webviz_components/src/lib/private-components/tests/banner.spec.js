import React from 'react';
import renderer from 'react-test-renderer';
import Banner from '../Banner';

describe('Banner', () => {
    it('should render Banner', () => {
        const tree = renderer.create(
            <Banner color="#333333" title="Banner Title" />
        );
        expect(tree.toJSON()).toMatchSnapshot();
    });
});
