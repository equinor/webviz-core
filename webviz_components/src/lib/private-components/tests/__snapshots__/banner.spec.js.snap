// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should render Banner 1`] = `
.c0 {
  background-color: #333333;
  height: 534px;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  color: #ffffff;
  font-size: 48px;
  font-family: var(--bannerFontFamily);
  font-weight: var(--bannerFontWeight);
}

@media (max-width:1200px) {
  .c0 {
    height: 380px;
  }
}

<div
  className="c0"
  color="#333333"
  role="banner"
>
  <p
    className="c1"
  >
    Banner Title
  </p>
</div>
`;
